website/docs/r/network_routed_v2.html.markdown

```diff
@@ -129,6 +130,55 @@ Static IP Pools support the following attributes:
 * `start_address` - (Required) The first address in the IP Range
 * `end_address` - (Required) The final address in the IP Range
 
+<a id="metadata"></a>
+## Metadata
```

Comment:
Just thinking, maybe there's a way to do "includes" in these docs, so we don't have to repeat the same section many times in different files?

Reply:
I think there's no built-in tool in markdown to do that, but we could do a pre-processing tool, pretty much like we do with the changelog, something like a template. We can discuss with @Didainius @dataclouder. I also felt the pain when doing this huge copypaste but as it was some kind of exception I didn't explore this option (also it would be out of scope of this PR).

I also thought about just linking to a new "metadata.md" under `guides`, but as it's also quite small and not really a guide, I also discarded this option.

Let's hear your thoughts!

Reply:
Thanks. It's fine for this PR, if there's no built-in way to do it.